packages/substrate/.env.test

```
@@ -0,0 +1 @@
+SYGMA_ENV="devnet"
```

```suggestion
SYGMA_ENV="testnet"
```

Super nitpick, but I would always default external devs to testnet.